Automattic/wp-calypso
# Help: initiate live chat from Inline Help

Right now, initiating a live chat session / contacting support is somewhat hidden on `/help` → "Contact us". This PR adds the contact form to Inline Help. 

Current status (with forced chat): 

![inline-help](https://user-images.githubusercontent.com/23619/36982116-55979c0a-208f-11e8-8bc4-bf6d34286288.gif)


Task list:

- [x] add a "Contact us" button to Inline Help
- [x] open the contact form when clicking the "Contact us" button
- [x] close the contact form when clicking the "Contact us" button again or when clicking the search input
- [x] make the contact form initiate an actual chat when submitting the form via "Chat with us", then reset and close the form and Inline Help
- [x] add an A/B test to be able to throttle volume
- [x] make sure we track usage
- [x] look into no selected site when starting from /me (d09ebfd)
- [x] make sure the different support variations work fine when initiated from the inline form
- [x] check sizes on mobile, especially height of the Popover
- [ ] merge Automattic/wp-e2e-tests/pull/1009 when merging this PR

To test: 
- make sure you're in the A/B test variant "inlinecontact" from the "inlineHelpWithContactForm" test (use the bottom right A/B test selection tool in development)
- make sure you can contact support from the inline help popover as shown in the GIF above
- make sure this works across plans and site types
- make sure this works on different screen sizes and devices
